seed/challenges/01-front-end-development-certification/html5-and-css.json

```diff
@@ -24,7 +24,10 @@
         "<h1>Hello</h1>"
       ],
       "tests": [
-        "assert.isTrue((/hello(\\s)+world/gi).test($('h1').text()), 'message: Your <code>h1</code> element should have the text \"Hello World\".');"
+        {
+          "text": "Your <code>h1</code> element should have the text \"Hello World\".');",
+          "testString": "assert.isTrue((/hello(\\s)+world/gi).test($('h1').text()), 'Your <code>h1</code> element should have the text \"Hello World\".');"
+        }
```

Comment:
This is an example of the new test object shape. It must have a text field and a testString field.